kasa/device_factory.py

```diff
@@ -128,7 +128,7 @@ def get_device_class_from_sys_info(info: Dict[str, Any]) -> Type[SmartDevice]:
 
 def get_device_class_from_family(device_type: str) -> Optional[Type[SmartDevice]]:
     """Return the device class from the type name."""
-    supported_device_types: dict[str, Type[SmartDevice]] = {
+    supported_device_types: Dict[str, Type[SmartDevice]] = {
```

Comment:
Local lint checks were failing due to this, fixing even though it's unrelated to the general PR contents.

Reply:
That's curious, but yeah, we are still using the container types from typing module to keep python3.8 support available. Thanks for the fix :+1: